### PR TITLE
(feature) Store list

### PR DESCRIPTION
**Pull request by dattran92**
- Also update merchant list
- Handle asset version to avoid cache every release
- Update some UI stuffs

### CODE REVIEW

File: init.php [hunk 1]
```diff
@@ -208,25 +230,37 @@ function bfx_crypto_map_handler( $atts ) {
...
+    }
+
+    // const mapboxKey = 'pk.eyJ1IjoiZGF0dHJhbmJmeCIsImEiOiJjbG5reXVoYjEwenF4MmlzMzlmOWhpZ3J6In0.y3REJgotRpiNyo_tYAx2yQ';
+
+    // const gl = L
+    //   .mapboxGL({
+    //     style: 'mapbox://styles/dattranbfx/clnaw7jkh01rl01qn3llp3wur',
+    //     accessToken: mapboxKey,
+    //   })
+    //   .addTo(map);
```
Comment (eugeneglova) [inline init.php:254]: do we need it to be commented?

File: init.php [hunk 2]
```diff
@@ -208,25 +230,37 @@ function bfx_crypto_map_handler( $atts ) {
...
 
     const tiles = L
       .tileLayer('https://tile.openstreetmap.org/{z}/{x}/{y}.png', {
         maxZoom: 19,
-        attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>'
+        attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>' ,
```
Comment (eugeneglova) [inline init.php]:
```suggestion
        attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>',
```

File: init.php [hunk 3]
```diff
@@ -337,30 +371,34 @@ function renderMarkers(data) {
...
+              [merchant.lat, merchant.lng],
+              {
+                merchantId: merchant.id,
+                icon: markerIcon,
+              },
+            )
+            .on('click', onMarkerClick);
+
+          if (!isMobile) {
+            marker.bindPopup('', popupOptions)
```
Comment (eugeneglova) [inline init.php]: just for consistency with other lines
```suggestion
            marker.bindPopup('', popupOptions);
```

File: init.php [hunk 4]
```diff
@@ -399,7 +429,31 @@ function filterMarkers() {
...
+      };
+    }
+
+    function filterMarkers() {
+      const filterData = getFilterData();
+      const numberOfFilter = filterData.numberOfFilter;
+      const filteredData = filterData.filteredData;
+
+      if (numberOfFilter > 0) {
+        jQuery('#filter-number').html(numberOfFilter + '').addClass('active');
```
Comment (eugeneglova) [inline init.php:445]: adding empty string to convert it to string ?

File: init.php [hunk 5]
```diff
@@ -467,12 +608,16 @@ function add_script_attributes( $html, $handle ) {
 function bfx_crypto_map_shortcode_scripts() {
   global $post;
   if ( is_a( $post, 'WP_Post' ) && has_shortcode( $post->post_content, 'bfx_crypto_map') ) {
+    $plugin_version = bfx_crypto_map_version();
     wp_enqueue_script('leaflet', 'https://unpkg.com/leaflet@1.9.4/dist/leaflet.js', array(), null);
     wp_enqueue_script('leaflet-marker-cluster', 'https://unpkg.com/leaflet.markercluster@1.5.3/dist/leaflet.markercluster.js', array('leaflet'), null);
+    // wp_enqueue_script('mapbox-gl', 'https://api.tiles.mapbox.com/mapbox-gl-js/v2.14.1/mapbox-gl.js', array(), null);
+    // wp_enqueue_script('mapbox-gl-leaflet', 'https://unpkg.com/mapbox-gl-leaflet/leaflet-mapbox-gl.js', array('leaflet', 'mapbox-gl'), null);
```
Comment (eugeneglova) [inline init.php:616]: do we need it to be commented?

File: init.php (hunk 1 above)
Comment (dattran92) [reply init.php:254]: this is about mapbox integration. I have done it and demo to Luis. He is happy with it. But as we are closed to Lugano event, so I need to revert the change to release the other stuffs. comment this for easier enabling it in the future.

File: init.php (hunk 4 above)
Comment (dattran92) [reply init.php:445]: Yes. I think it's safer than `.toString()` because we don't worry about `null.toString()` exception

File: init.php (hunk 5 above)
Comment (dattran92) [reply init.php:616]: Yes. for easier enabling mapbox integration in the next release.

**Comment by dattran92** [comment]
Thanks @eugeneglova for reviewing:
- Commit changes as you suggested (haven't added any linting things to the project, so that would be things I would definitely do in the future).
- Added explanation for the other things.

**Review by eugeneglova — APPROVED** [review]

**Pull request merged**